### PR TITLE
Banner updates on Home page, Community and SMP pages

### PR DESCRIPTION
**Pull request by debo19**
Signed-off-by: Debopriya Bhattacharjee <debopriyabhattacharjee@Debopriyas-MacBook-Air.local>

**Description**
1) Removed MeshMap banner video from mobile view. Added a background overlay instead.
2) Changed a couple of `SEO` titles to remove occurrences of `service mesh company` etc.
3) Added the banner image optimization present on the home page to the community and smp pages.

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: src/collections/blog/2021/2021-10-01-pipelining-service-mesh-specifications/index.mdx [hunk 1]
```diff
@@ -29,7 +29,7 @@ import githubBlack from "../../../../assets/images/socialIcons/github_black.svg"
   
 <p>With growing adoption of service meshes in cloud native environments, service mesh abstractions - service mesh-neutral specifications - have emerged. <Link to="/projects/service-mesh-performance">Service Mesh Performance</Link>  and <Link to="/projects/service-mesh-interface-conformance">Service Mesh Interface</Link> are two open specifications that address the need for universal interfaces for interacting with and managing any type of service mesh. Let’s examine what each specification provides.</p>
 
-<img src={smpLogo} className="image-left-no-shadow" alt="service mesh performance logo" /><p><Link to="https://smp-spec.io">Service Mesh Performance</Link> standardizes service mesh value measurement, characterizing any deployment's performance by capturing the details of infrastructure capacity, service mesh configuration and workload metadata.</p>
```
Comment (leecalcote) [inline src/collections/blog/2021/2021-10-01-pipelining-service-mesh-specifications/index.mdx:32]: good catch here.

File: src/pages/blog/index.js [hunk 2]
```diff
@@ -75,8 +75,7 @@ const Blog = (props) => {
         <GlobalStyle />
         <SEO
           title="Blog"
-          description="Articles how to service mesh from the world's largest service mesh community.
-              Service mesh how-tos and cloud native ecosystem news."
+          description="Share your knowledge, experiences or latest news through the Layer5 blog."
```
Comment (leecalcote) [inline src/pages/blog/index.js:78]:
```suggestion
          description="The latest news and announcements about Layer5, our products, and our ecosystem, as well as voices from across our community."
```

This isn't about contributors, but is about users.

File: src/pages/community/community-managers.js [hunk 3]
```diff
@@ -16,7 +16,7 @@ const ManagersPage=() => {
       <Layout>
         <GlobalStyle />
         <SEO title="Managers"
-          description="Managers - A Community Manager is a person who has an innate drive to contribute to the community's prosperity."
+          description="A Community Manager is a person who has an innate drive to contribute to the community's prosperity."
```
Comment (leecalcote) [inline src/pages/community/community-managers.js:19]:
```suggestion
          description="Layer5 Community Managers have an innate drive to contribute to the community's prosperity and that of our individual contributors' prosperity."
```

File: src/pages/community/handbook/writing-program.js [hunk 4]
```diff
@@ -15,7 +15,7 @@ const Faq = () => (
   <ThemeProvider theme={theme}>
     <Layout>
       <GlobalStyle />
-      <SEO title="Writing Program" description="The Layer5 Writing Program is a way to demonstrate your expertise, give back to the community, and help us produce great content." />
+      <SEO title="Writing Program" description="We work with authors and technologists from around the world to produce high quality articles on cloud native management, Kubernetes, service meshes and related technologies." />
```
Comment (leecalcote) [inline src/pages/community/handbook/writing-program.js:18]:
```suggestion
      <SEO title="Writing Program" description="Work with authors and technologists from around the world to share your knowledge of cloud native infrastructure, modern application management, Kubernetes, service meshes and related technologies." />
```

File: src/pages/community/members.js [hunk 5]
```diff
@@ -185,7 +185,7 @@ const MembersPage = () => {
         <GlobalStyle />
         <SEO
           title="Members"
-          description="Members - The Layer5 contributors list"
+          description="The Layer5 contributors list"
```
Comment (leecalcote) [inline src/pages/community/members.js:188]:
```suggestion
          description="An awarding-winning, open source community with a warm and welcoming collection of contributors."
```

File: src/pages/company/legal/terms-of-service.js [hunk 6]
```diff
@@ -17,7 +17,7 @@ const TermsOfService = () => (
       <GlobalStyle />
       <SEO title="Terms of Service" description="Contact Layer5 for help with operating a service mesh.
```
Comment (leecalcote) [inline src/pages/company/legal/terms-of-service.js:18]:
```suggestion
      <SEO title="Terms of Service" description="Layer5's cloud native application and infrastructure management software enables organizations to expect more from their infrastructure.
```

File: src/pages/company/legal/terms-of-service.js [hunk 7]
```diff
@@ -17,7 +17,7 @@ const TermsOfService = () => (
...
   Layer5 is the makers of Meshery and service mesh standards.
-  We are the largest collection of service mesh projects and their maintainers in the world." />
+  We are the largest collection of cloud native projects and their maintainers in the world." />
```
Comment (leecalcote) [inline src/pages/company/legal/terms-of-service.js:20]:
```suggestion
" />
```

File: src/pages/index.js [hunk 8]
```diff
@@ -47,7 +47,7 @@ const IndexPage = () => {
   return (
     <ThemeProvider theme={theme}>
       <GlobalStyle />
-      <SEO title="Layer5" description="Making service meshes available to the rest of us. Open source software for management of service meshes. Allowing developers to focus on business logic, not infrastructure concerns. Empowering operators to confidentally run modern infrastructure."
+      <SEO title="Layer5" description="An empowerer of engineers, Layer5 helps you extract more value from your infrastructure. Creator and maintainer of service mesh standards. Maker of Meshery, the cloud native management plane."
```
Comment (leecalcote) [inline src/pages/index.js:50]: Nice!

File: src/pages/learn/learning-paths.js [hunk 9]
```diff
@@ -14,7 +14,7 @@ const LearningPathsPage = () => {
     <ThemeProvider theme={theme}>
       <Layout>
         <GlobalStyle />
-        <SEO title="Learn Service Mesh"
+        <SEO title="Learning Paths"
```
Comment (leecalcote) [inline src/pages/learn/learning-paths.js:17]:
```suggestion
        <SEO title="DevOps, GitOps, and Cloud Native Learning Paths"
```

File: src/pages/learn/service-mesh-labs.js [hunk 10]
```diff
@@ -25,7 +25,7 @@ const ServiceMeshLabsPage = (props) => {
     <ThemeProvider theme={theme}>
       <Layout>
         <GlobalStyle />
-        <SEO title="Interactive Service Mesh Labs" description="Learn to service mesh with the industry leaders.Try interactive service mesh lab scenarios." />
+        <SEO title="Interactive Labs" description="Learn to service mesh with the industry leaders.Try interactive service mesh lab scenarios." />
```
Comment (leecalcote) [inline src/pages/learn/service-mesh-labs.js:28]:
```suggestion
        <SEO title="Interactive DevOps, GitOps, and Cloud Native Labs" description="Learn to manage Kubernetes and Istio with experts from Layer5. Try an interactive service mesh lab." />
```

File: src/pages/learn/service-mesh-workshops.js [hunk 11]
```diff
@@ -14,7 +14,7 @@ const WorkshopsGridPage = () => {
     <ThemeProvider theme={theme}>
       <Layout>
         <GlobalStyle />
-        <SEO title="Service Mesh Workshops" description="Service mesh workshops and tutorials for Istio, Envoy, Linkerd, Consul, App Mesh, Open Service Mesh" />
+        <SEO title="Workshops" description="Workshops and tutorials for Istio, Envoy, Linkerd, Consul, App Mesh, Open Service Mesh" />
```
Comment (leecalcote) [inline src/pages/learn/service-mesh-workshops.js:17]:
```suggestion
        <SEO title="Service Mesh Workshops" description="Service mesh workshops and tutorials for Istio, Envoy, Linkerd, Consul, App Mesh, Open Service Mesh, Cilium, Kuma, NGINX" />
```

File: src/pages/resources/index.js [hunk 12]
```diff
@@ -41,7 +41,7 @@ const MembersPage = () => {
     <ThemeProvider theme={theme}>
       <Layout>
         <GlobalStyle />
-        <SEO title="Service Mesh Resources" description="Articles on how to service mesh from the world's largest service mesh community. Service mesh how-tos and cloud native ecosystem news." canonical="https://layer5.io/resources" />
+        <SEO title="Cloud Native Resources" description="Learn how to provision, secure, connect, and manage your applications on Kubernetes and any service mesh. Service mesh how-tos and cloud native ecosystem news." canonical="https://layer5.io/resources" />
```
Comment (leecalcote) [inline src/pages/resources/index.js:44]:
```suggestion
        <SEO title="Cloud Native Resources" description="Learn how to provision, secure, connect, and manage your applications with GitOps and DevOps techniques running on Kubernetes and any service mesh. Service mesh how-tos and cloud native ecosystem tips and tricks." canonical="https://layer5.io/resources" />
```

File: src/templates/blog-tag-list.js [hunk 13]
```diff
@@ -48,7 +48,7 @@ const BlogListPage = ({ pageContext, data }) => (
       <GlobalStyle />
       <SEO
         title="Blog"
-        description="Articles how to service mesh from the world's largest service mesh community. Service mesh how-tos and cloud native ecosystem news."
+        description="Share your knowledge, experiences or latest news through the Layer5 blog."
```
Comment (leecalcote) [inline src/templates/blog-tag-list.js:51]:
```suggestion
        description="The latest news and announcements about Layer5, our products, and our ecosystem, as well as voices from across our community."
```

**Comment by l5io** [comment]
🚀 Preview for commit b6cd0d811e1af6bc80ace8af429b0147c7843cbf at: https://62e47253c322be1c28055c9d--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit bdce478b2a2fc5e510395c0b58ae17231bd558cf at: https://62e6de69fb289d1bfc740f18--layer5.netlify.app

**Comment by leecalcote** [comment]
Good!

**Pull request merged**